finland/router-config.json

```diff
@@ -5,8 +5,7 @@
     "elevatorBoardTime": 60,
     "bicycle": {
       "boardCost": 120,
-      "reluctance": 1.7,
-      "optimization": "safest-streets"
```

Comment:
Why is this removed from the defaults? I think it should be kept in the routing defaults, but there shouldn't be a need to configure it for the cache requests again with the same value.

Reply:
I think there was some kind of misunderstanding, I added it back